editor/components/block-list/block.js

```diff
@@ -286,10 +290,11 @@ export class BlockListBlock extends Component {
 	 * @see https://developer.mozilla.org/en-US/docs/Web/Events/mouseenter
 	 */
 	maybeHover() {
-		const { isMultiSelected } = this.props;
+		const { isMultiSelected, isSelected } = this.props;
 		const { isHovered } = this.state;
 
-		if ( isHovered || isMultiSelected || this.hadTouchStart ) {
+		if ( isHovered || isMultiSelected || isSelected ||
+				this.props.isMultiSelecting || this.hadTouchStart ) {
```

Comment:
Minor, but the other props are extracted via destructuring assignment, while `isMultiSelecting` isn't.

Reply:
>Minor, but the other props are extracted via destructuring assignment, while `isMultiSelecting` isn't.

Our good friend [`no-shadow`](https://eslint.org/docs/rules/no-shadow) doesn't like the destructuring 😄